python/src/messages.py

```diff
@@ -0,0 +1,3 @@
+from _messages import *
+
+ExpressionType = Type1
```

Comment:
I don't understand what this file does. Can you explain?

Reply:
We have two entities in the original model named Type (design bug from my perspective). This module is a simple adapter, so the end user will import Type and ExpressionType but not Type and Type1. In the serialized model they both are named Type as it was in the original model

Reply:
Done